### PR TITLE
prepare ui in modal for banned not banned member

### PR DESCRIPTION
**Pull request by krawczykkatarzyna**
Screen Shot 2023-07-07 at 18 42 37

### CODE REVIEW

File: src/components/v5/common/Fields/Switch/Switch.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.toggle::after {
+  @apply absolute top-0.5 left-0.5 bg-base-white rounded-full h-4 w-4 transition shadow-sm;
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Fields/Switch/Switch.module.css:2]: Please, use tailwind rules. You have an example i src/components/v5/common/Checkbox/Checkbox.module.css how you can do it ('peer' rule) 

https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state

File: src/components/v5/common/Fields/Switch/Switch.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import styles from './Switch.module.css';
+import { SwitchProps } from './types';
+
+const displayName = 'v5.common.Fields.Switch';
+
+const Switch: FC<SwitchProps> = ({ id = 'switch-id', isDisabled }) => (
+  <label
+    htmlFor={id}
+    className={clsx('relative', {
+      'cursor-not-allowed': isDisabled,
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Fields/Switch/Switch.tsx:13]: As I can see we used 'pointer-events-none' so far for the disabled state. Please, correct it to be consistent

File: src/components/v5/common/Fields/Switch/Switch.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      'cursor-pointer': !isDisabled,
+    })}
+  >
+    <input type="checkbox" id={id} className="sr-only" disabled={isDisabled} />
+    <div
+      className={`${styles.toggle} bg-gray-200 border-2 border-gray-200 h-5 w-9 rounded-full`}
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Fields/Switch/Switch.tsx:19]: You can use 'clsx' library here

File: src/components/v5/common/Fields/Switch/Switch.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+const Switch: FC<SwitchProps> = ({ id = 'switch-id', isDisabled }) => (
+  <label
+    htmlFor={id}
+    className={clsx('relative', {
+      'cursor-not-allowed': isDisabled,
+      'cursor-pointer': !isDisabled,
+    })}
+  >
+    <input type="checkbox" id={id} className="sr-only" disabled={isDisabled} />
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Fields/Switch/Switch.tsx:17]: What does 'sr-only' mean?

File: src/components/v5/common/Fields/Textarea/Textarea.tsx [hunk 5]
```diff
@@ -61,10 +67,10 @@ const Textarea: FC<TextareaProps> = ({
         <textarea
           id="message"
           placeholder={placeholder}
-          className="resize-none w-full md:min-w-[32rem] text-gray-900 text-md outline-0 placeholder:text-gray-500"
+          className="resize-none w-full text-gray-900 text-md outline-0 placeholder:text-gray-500"
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Fields/Textarea/Textarea.tsx:70]: Do we need text-gray-900 class here?

File: src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx [hunk 6]
```diff
@@ -44,6 +54,44 @@ const ManageMemberModal: FC<ManageMemberModalProps> = ({
         placeholderText={{ id: 'members.modal.selectActions' }}
         isListRelative
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx:55]: What does ' isListRelative' prop mean? I haven't seen it earlier. I think this name isn't correct.

File: src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx [hunk 7]
```diff
@@ -44,6 +54,44 @@ const ManageMemberModal: FC<ManageMemberModalProps> = ({
...
+              <span className="text-gray-400 flex">
+                <Icon name="info" appearance={{ size: 'extraTiny' }} />
+              </span>
+            </div>
+            <Switch />
+          </div>
+
+          <Textarea textareaTitle={{ id: 'members.ban.textarea.label' }} />
+
+          <div className="flex mt-8 gap-2">
```
Comment (KatarzynaStefanowiczPagepro) [inline src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx:85]: As I can see we have a login to Cancel and Confirm buttons in the Modal component. There are props closeMessage and confirmMessage. Please, use them to be consistent and not repeat buttons and their styles.

File: src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx (hunk 6 above)
Comment (CzarekDryl) [reply src/components/v5/common/Modals/ManageMemberModal/ManageMemberModal.tsx:55]: This prop changes position of list in select to relative from absolute.

**Review by CzarekDryl — APPROVED** [review]

**Pull request merged**